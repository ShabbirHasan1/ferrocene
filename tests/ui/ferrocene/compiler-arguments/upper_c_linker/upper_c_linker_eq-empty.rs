// When linker is specified, but empty, we have a linker error, so we get it
// on the build step.
//
// build-fail
// compile-flags: -Clinker=

fn main() {}

// ferrocene-annotations: um_rustc_C_linker
