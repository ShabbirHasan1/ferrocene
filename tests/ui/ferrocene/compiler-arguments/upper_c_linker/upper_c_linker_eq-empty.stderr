error: couldn't extract file stem from specified linker

error: aborting due to previous error

